modules/backend/assets/js/october.treelist.js

```diff
@@ -46,7 +46,7 @@
     }
 
     TreeListWidget.prototype.dispose = function() {
-        this.unbind()
+        this.off()
```

Comment:
What's the rationale for changing this? Especially when unbind() is defined as a function on this object.

Reply:
I will link you two things:

1. bind() - https://github.com/jquery/jquery-migrate/blob/master/warnings.md#jqmigrate-jqueryfnbind-is-deprecated

2. reason: https://github.com/jquery/jquery-migrate/blob/master/warnings.md#jqmigrate-jqueryfnclick-event-shorthand-is-deprecated